saltyrtc/message.py

```diff
@@ -1,30 +1,33 @@
 import abc
-import struct
-import io
 import binascii
+import io
+import struct
 
 # noinspection PyPackageRequirements
 import umsgpack
 
-from .exception import *
+from .common import sign_keys as sign_keys_
```

Comment:
What does the syntax with the `_` suffix? I think I've never seen that in Python code :)